charts/capsule-proxy/templates/certgen-job.yaml

```diff
@@ -5,8 +5,14 @@ metadata:
   name: {{ include "capsule-proxy.fullname" . }}-certgen
   labels:
     {{- include "capsule-proxy.labels" . | nindent 4 }}
+  {{- with .Values.jobs.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
 spec:
-  ttlSecondsAfterFinished: 60
+  {{- if ge .Values.jobs.ttlSecondsAfterFinished 0.0 }}
+  ttlSecondsAfterFinished: {{ .Values.jobs.ttlSecondsAfterFinished }}
+  {{- end }}
```

Comment:
Are you sure `ge` is the right operator here?

Otherwise, we'll have the `ttlSecondsAfterFinished` value to zero, and according to the documentation:

```
If this field is set to zero, the Job becomes eligible to be deleted immediately after it finishes.
```

Reply:
Yeah, maybe somebody intends to do that?  
As of `0` is a valid value for the spec, I would make it possible to set it.